pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -36,10 +37,18 @@ type EventListenerSpec struct {
 //
 // +k8s:deepcopy-gen=true
 type Trigger struct {
+	TriggerValidate *TriggerValidate   `json:"validate,omitempty"`
```

Comment:
For consistency, I think it makes sense to use a struct rather than a pointer.

Reply:
I did this because this is optional. I was using the pointer to check whether we want to validate or not.